### PR TITLE
Pull Request: Milestone 2 (Components and Store)

### PR DESCRIPTION
**Pull request by johnsonsirv**
### In this PR I achieved the following
**_1. Prepared the directory structure_**
 - Create a components directory in the src folder for all the presentational components.
 - Create a containers directory in the src folder for all the container components.
 - Create a actions directory with the index.js file where you’ll define all the Redux actions.
 - Create a reducers directory with two files: books.js and index.js. The former will store the functions responsible for actions on Books, and the latter should combine all the reducers into one. For now, it will only combine the books.js reducer.

**_2. Added Presentational & Container Components_**
 - Implement the **App** component (presentational)
 - Implement the **BooksList** component (container)
 - Implement the **Book** component (presentation)
 - Implement the **BooksForm** component(container)
>> Added extra **Button** component to render the ``Remove Button``

**_3. Connect App to Redux Store_**
 - Define the Redux actions and reducers
 - Initialize the Redux store

**_This milestone was fun to build_**

### PS: I am working Solo, and Student Succes team is Aware

**_Thanks in Advance! Happy Reviewing :sunglasses:_**

### CODE REVIEW

File: src/containers/bookForm.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const BookForm = () => {
```
Comment (JamesOkunlade) [inline src/containers/bookForm.jsx:3]: `BookForm` should be a stateful component and its state should be initialized with `title` and `category`.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# ♻️ Changes required

**Really good job so far.** 

One thing that should implement to get a pass on this milestone is to make `BookForm` a stateful component and initialize it with `title` and `category`.

Request for re-review when you're done.

**Goodluck** 😁 
[James](https://twitter.com/JamesOkunlade)

**Comment by johnsonsirv** [comment]
Thank you @JamesOkunlade for the review.

This PR is for **Milestone 2** and according to the requirements [Here](https://github.com/microverseinc/project-redux-bookstore/blob/master/milestones/MILESTONE_2.md) it involves setting up components and connecting them to store.

**Milestone 3** will make ```BookForm``` stateful as started [Here](https://github.com/microverseinc/project-redux-bookstore/blob/master/milestones/MILESTONE_3.md)

Kindly re-review. Thank You.

**Review by JamesOkunlade — APPROVED** [review]
# Approved 😁 ✅ 

**I'm sorry @johnsonsirv  for misleading you 😁** 

The requirements on pathwright differ from what I have on my guideline and that's why, we'll make necessary corrections.

Please merge and move on to the next milestone as it meets the minimum requirements.

**Goodluck** 🙂 
[James](https://twitter.com/JamesOkunlade)

**Pull request merged**